wiedza_i_zycie/ui/src/app/articles/articles.component.html

```diff
@@ -0,0 +1,74 @@
+
+<mat-sidenav-container>
+    <mat-sidenav
+      #sidenav
+      mode="over"
+    >
+    <button
+      mat-icon-button
+      (click)="sidenav.toggle()"
+    >
+    <mat-icon>arrow_forward</mat-icon>
+    </button>
+  
+      <br>
+      <br>
+      <br>
```

Comment:
nie używaj proszę `<br>` to ustaleniu układu strony jest to antypattern. Do wykładu strony użyj proszę css